### PR TITLE
Petersburg

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**

### CODE REVIEW

File: new/PRS12786Yohannes.xml [hunk 1]
```diff
@@ -61,7 +61,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <floruit/>
                 </person>
                 <listRelation>
-                    <relation name="saws:isRelatedTo" active="PRS12786Yohannes" passive=""/>
+                    <relation name="saws:isRelatedTo" active="PRS12786Yohannes" passive="RNBefns7"/>
```
Comment (thea-m) [inline new/PRS12786Yohannes.xml]: isAttested would maybe be a better relation here?

File: new/PRS12787GabraEgziabeher.xml [hunk 2]
```diff
@@ -48,18 +50,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Gabra ʾƎgziʾabǝḥer</persName>
                     <faith type="EOTC"/>
                     <occupation type="other"/>
                     <birth/>
                     <death/>
                     <floruit/>
                 </person>
                 <listRelation>
-                    <relation name="saws:isRelatedTo" active="PRS12787GabraEgziabeher" passive=""/>
+                    <relation name="saws:isRelatedTo" active="PRS12787GabraEgziabeher" passive="RNBefsn7"/>
```
Comment (thea-m) [inline new/PRS12787GabraEgziabeher.xml]: maybe better isAttestedIn

File: new/PRS12787GabraEgziabeher.xml [hunk 3]
```diff
@@ -30,7 +30,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p/>
+                <p>Gabra ʾƎgziʾabǝḥer is mentioned as
```
Comment (thea-m) [inline new/PRS12787GabraEgziabeher.xml]: I would add this information either as a desc in the relation, or in occupation

File: new/PRS12858WalattaGabreel.xml [hunk 4]
```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS12858WalattaGabreel" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>WalattaGabreel</title>
```
Comment (thea-m) [inline new/PRS12858WalattaGabreel.xml]: same as above

File: new/PRS12858WalattaGabreel.xml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Depicted as donor in <ref type="mss" corresp="IVef17"/></p>
```
Comment (thea-m) [inline new/PRS12858WalattaGabreel.xml]: see above

File: new/PRS12858WalattaGabreel.xml [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1"> ወለተ፡ ገብርኤል፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walatta Gabrǝʾel</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="other"/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
```
Comment (thea-m) [inline new/PRS12858WalattaGabreel.xml]: empty elements  could be deleted

File: new/PRS12858WalattaGabreel.xml [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+                </person>
+                <listRelation>
+                    <relation name="saws:isRelatedTo" active="PRS12858WalattaGabreel" passive="IVef17"/>
```
Comment (thea-m) [inline new/PRS12858WalattaGabreel.xml]: see above

**Comment by thea-m** [comment]
I meant the relation lawd:hasAttestation, sorry

File: new/PRS12787GabraEgziabeher.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/PRS12787GabraEgziabeher.xml]: well, he is rather (actively)  related then mentioned. For me, a person who is mentioned in the text as protagonists would be "mentioned".

File: new/PRS12787GabraEgziabeher.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/PRS12787GabraEgziabeher.xml]: Ok

File: new/PRS12858WalattaGabreel.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply new/PRS12858WalattaGabreel.xml]: Ok

File: new/PRS12858WalattaGabreel.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply new/PRS12858WalattaGabreel.xml]: Ok

File: new/PRS12858WalattaGabreel.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply new/PRS12858WalattaGabreel.xml]: Ok

File: new/PRS12858WalattaGabreel.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply new/PRS12858WalattaGabreel.xml]: Same as above.

File: new/PRS12786Yohannes.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS12786Yohannes.xml]: No.

**Comment by PietroLiuzzo** [comment]
As I did not know what the problem was with those three files I made a local copy and removed them from this PR. I will now merge this and make a new PR for those files.

**Comment by thea-m** [comment]
I still see inconsistency in the use of saws:isRelatedTo, also in the files now merged, see https://github.com/BetaMasaheft/Documentation/issues/1307

**Comment by PietroLiuzzo** [comment]
Sorry @thea-m ,  I know that the review was not successfully completed, I wanted however to solve the conflict. It looks like perhaps this will have to be addressed with a dedicated set of edits and a PR perhaps. Thank you

**Comment by thea-m** [comment]
Yes yes, this was getting too confusing. It can be solved through the issue. Thank you!

**Pull request merged**